docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    volumes:
+      - pgdata:/var/lib/postgresql/data
```

Comment:
Why are you mounting pgdata => /var/lib/postgresql/data on app?. This should be on db instead. Also please update gitignore to include `pgdata` directory.

Reply:
Ref: https://docs.docker.com/compose/environment-variables/